docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -10,225 +10,77 @@ images: ["/installation/thumbnails/esp32-espressif.png"]
-cd ~/esp/esp-idf
-./install.sh esp32
-```
+- [Linux (x86_64)](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-amd64-linux)
+- [Linux (Aarch64)](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-arm64-linux)
+- [MacOS](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-macos)
+- [Windows](https://github.com/viamrobotics/micro-rdk/releases/latest/download/micro-rdk-installer-windows.exe)
 
-To activate ESP-IDF, run the following command to source (`.`) the activation script `export.sh`:
+### Flash your ESP32 with the Micro RDK Installer
 
-```sh { class="command-line" data-prompt="$"}
-. $HOME/esp/esp-idf/export.sh
-```
+Navigate to [the Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot) in your desired location.
 
-{{< alert title="Tip" color="tip" >}}
-To avoid conflicts with other toolchains, adding this command to your `.bashrc` or `.zshrc` is not recommended.
-Instead, save this command to run in any future terminal session where you need to activate the ESP-IDF development framework.
-{{< /alert >}}
+1. Click on the name of the robot to go to the robot's page.
+2. Click on the **Setup** tab.
+3. Regardless of your operating system, select **Mac** and press the button that appears in Step 2 to download the Viam app config for your robot.
 
-#### Install the Rust ESP Toolchain and Activate the ESP-RS Virtual Environment
+   In your terminal, `cd` to the directory where you downloaded your pre-built binary and run the following command to flash the micro-RDK directly to an ESP32 connected to your computer through a data cable:
 
-First, install the following tools with `cargo`:
+   {{< tabs name="Download pre-built binaries" >}}
+   {{% tab name="Linux (x86_64)"%}}
 
 ```sh { class="command-line" data-prompt="$"}
-cargo install espup
+./micro-rdk-installer-amd64-linux write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-```sh { class="command-line" data-prompt="$"}
-cargo install cargo-espflash v2.0.0-rc.1
-```
+    {{% /tab %}}
+    {{% tab name="Linux (Aarch64)" %}}
 
 ```sh { class="command-line" data-prompt="$"}
-cargo install ldproxy
+./micro-rdk-installer-arm64-linux write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-To download and install the esp-rs toolchain, run:
+    {{% /tab %}}
+    {{% tab name="MacOS" %}}
 
 ```sh { class="command-line" data-prompt="$"}
-espup install -s -f ~/esp/export-rs.sh -v 1.67.0
+./micro-rdk-installer-macos write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-To activate the ESP Rust toolchain, run the following command to source (`.`) the activation script `export-rs.sh`:
+    {{% /tab %}}
+    {{% tab name="Windows" %}}
 
 ```sh { class="command-line" data-prompt="$"}
-. $HOME/esp/export-rs.sh
+./micro-rdk-installer-windows write-flash --app-config=<your-file-path-to/viam.json>
```

Comment:
`./micro-rdk-installer-windows.exe`. That file suffix is Windows specific

Reply:
Sounds good, done